src/components/inputBoolean/AposBooleanInput.stories.js

```diff
@@ -34,29 +34,29 @@ storiesOf('Inputs (Boolean)', module)
     },
     template: `<AposBooleanInput :field="field" :value="value" :status="status"/>`
   }))
-  .add('Boolean (prechecked)', () => ({
-    components: { AposBooleanInput },
-    data () {
-      return {
-        field,
-        status: {},
-        value: {
-          data: false
-        }
-      }
-    },
-    template: `<AposBooleanInput :field="field" :value="value" :status="status"/>`
-  }))
-  .add('Toggle', () => ({
-    components: { AposBooleanInput },
-    data () {
-      return {
-        field: toggleField,
-        status: {},
-        value: {
-          data: false
-        }
-      }
-    },
-    template: `<AposBooleanInput :field="field" :value="value" :status="status"/>`
-  }))
+  // .add('Boolean (prechecked)', () => ({
```

Comment:
Why are you commenting these out? I think the last one is because you decided not to go with the labeled boolean, but we want a pre-checked version, right?

Reply:
i brought it back